scripts/fix-links.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+
+const fs = require('fs');
+const { platform } = require('os');
+
+const { processData } = require('../test/linter/test-links.js');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
+
+/**
+ * @param {Promise<void>} filename
+ */
+const fixLinks = (filename) => {
+  let errors = processData(filename);
+  let original = fs.readFileSync(filename, 'utf-8').trim();
```

Comment:
Guess ESLint would complain about this:

```suggestion
  const errors = processData(filename);
  const original = fs.readFileSync(filename, 'utf-8').trim();
```